data/extensions/Spells/Commands/InfoCommand.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using NeoServer.Game.Combat.Spells;
+using NeoServer.Game.Common;
+using NeoServer.Game.Common.Contracts.Creatures;
+using NeoServer.Game.Common.Contracts.Items;
+using NeoServer.Game.Common.Contracts.Items.Types;
+using NeoServer.Game.Common.Creatures;
+using NeoServer.Game.Items;
+using NeoServer.Networking.Packets.Outgoing;
+using NeoServer.Server.Common.Contracts;
+using NeoServer.Server.Helpers;
+
+namespace NeoServer.Extensions.Spells.Commands;
+
+public class InfoCommand : CommandSpell
+{
+    public override bool OnCast(ICombatActor actor, string words, out InvalidOperation error)
+    {
+       error = InvalidOperation.NotPossible;
+        
+       if (Params.Length != 1)
+            return false;
+
+       var ctx = IoC.GetInstance<IGameCreatureManager>();
+       ctx.TryGetPlayer(Params[0].ToString(), out var target);
+
+       if (target is null)
+           return false;
+       
+       var info = CreateItemInfo();
+       
+       var message =
+           $"""
+            AccountId: {target.AccountId}
+            Position: {target.Location.X}, {target.Location.Y}, {target.Location.Z}
+            Capacity: {target.TotalCapacity}
+            PremiumTime: {target.PremiumTime}
+            Level: {target.Level}
+            Skills:
+            {target.Skills.Where(item => item.Key != SkillType.Level).Select(Item =>   "   * " + Item.Key + ": " + Item.Value.Level + "\n").Aggregate((a, b) => a + b)}
+            """;
+
+       var window = new ListCommandsCommand.TextWindow(info, target.Location, message);
+       
+       window.WrittenBy = "OpenCoreMMO - SERVER";
```

Comment:
perhaps would be better to not have this text hard coded. Any chance to add on env ?

Reply:
Everything is possible add on env, but what would be good name for this variable?

Reply:
Lets catch the server name from appsettings and concat in the string:

![image](https://github.com/caioavidal/OpenCoreMMO/assets/11817846/5dd6a2db-a80e-42a7-8770-877b0bf6d9d3)